doxygen-awesome.css

```diff
@@ -1791,11 +1795,18 @@ table.memberdecls .memItemRight  {
 }
 
 table.memberdecls .mdescLeft, table.memberdecls .mdescRight {
-    background: none;
+    background: var(--fragment-desc-background);
     color: var(--page-foreground-color);
     padding: var(--spacing-small) 0;
 }
 
+table.memberdecls .mdescLeft{
```

Comment:
alright, the small gap is no longer there, i was going to remove bottom border for `memItemL/R`, but it would look a bit weird when there's no `mdescL/R`, so that's why i left it as is.